nota-studios/src/app/(pages)/services/page.tsx

```diff
@@ -1,26 +1,24 @@
 import ServicesList from "@/app/components/services-accordian-list";
-import SessionButton from "@/app/components/book-session-button";
+import PageH1 from "@/app/components/global/page-h1";
 import { STUDIO_HOURS, STUDIO_EQUIPMENT } from "@/app/constants/constants";
 
 export default function Services(){
     return(
         <>
-            <div className="flex md:flex-row sm:flex-col">
-                <div className="md:w-1/2 sm:w-full p-6 md:order-2" >
+            <PageH1 title="Services"/>
+            <div className="flex md:flex-row flex-col">
+                <div className="md:w-1/2 sm:w-full px-6 pb-6 md:order-2" >
                     <ServicesList />
-                    <div className="w-full mt-10 h-4 md:order-3">
-                        <SessionButton />
-                    </div>
                 </div>
-                <div className="md:w-1/2 sm:w-full p-6 md:order-first">
+                <div className="md:w-1/2 w-full p-6 md:order-first text-center md:text-left">
                     <h3>Studio Hours</h3>
                     {STUDIO_HOURS.map((schedule, index) =>(
```

Comment:
I see what you meant here about the .map() method. 

I just checkout out some docs on it. Is this right?
[https://www.w3schools.com/jsreF/jsref_map.asp](https://www.w3schools.com/jsreF/jsref_map.asp )

Reply:
Yes this is what I was looking into.